src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1114,7 +1114,11 @@ namespace ReportSizingManager {
 						}
 					}
 				} else if ( SizingType == HeatingWaterDesCoilLoadUsedForUASizing ) {
-					if ( TermUnitSingDuct || TermUnitPIU || TermUnitIU ) {
+					if ( TermUnitSingDuct ) {
+						Cp = GetSpecificHeatGlycol( PlantLoop( DataWaterLoopNum ).FluidName, DataGlobals::HWInitConvTemp, PlantLoop( DataWaterLoopNum ).FluidIndex, CallingRoutine );
+						rho = GetDensityGlycol( PlantLoop( DataWaterLoopNum ).FluidName, DataGlobals::CWInitConvTemp, PlantLoop( DataWaterLoopNum ).FluidIndex, CallingRoutine );
+						AutosizeDes = DataWaterFlowUsedForSizing * DataWaterCoilSizHeatDeltaT * Cp * rho;
```

Comment:
Yeah, I mean this appears reasonable, but because of the really chunky changes to sizing coming out as a result of this change, I'll need to learn more about `DataWaterFlowUsedForSizing` before merging.  I'm afraid the change in sized values will be alarming to many users.